mlflow/pyfunc/model.py

```diff
@@ -391,19 +396,254 @@ def predict_stream(
 @experimental
 class ChatAgent(PythonModel, metaclass=ABCMeta):
     """
-    A subclass of :class:`~PythonModel` that makes it more convenient to implement agents
-    that are compatible with popular LLM chat APIs. By subclassing :class:`~ChatAgent`,
-    users can create MLflow models with a ``predict()`` method that is more convenient
-    for chat tasks than the generic :class:`~PythonModel` API. ChatAgents automatically
-    define input/output signatures and an input example, so manually specifying these values
-    when calling :func:`mlflow.pyfunc.save_model() <mlflow.pyfunc.save_model>` is not necessary.
+    **What is the ChatAgent Interface?**
+
+    The ChatAgent interface is a chat schema specification that has been designed for advanced
```

Comment:
```suggestion
    The ChatAgent interface is a chat schema specification that has been designed for authoring conversational agents.
```

Better not to say advanced, since for less advanced use cases ChatAgent works too